libasynql/src/poggit/libasynql/base/SqlSlaveThread.php

```diff
@@ -96,7 +97,7 @@ public function run() : void{
 	/**
 	 * @return bool
 	 */
-	public function isBusy(): bool {
+	public function isBusy() : bool{
```

Comment:
This change is irrelevant to this pull request.

Consider setting "Only VCS changed text" in your phpstorm settings.

How spaces are placed here is not a big deal since they do not affect readability a lot (just OCD). But it is a problem if the git blame is corrupted by irrelevant changes.